db_release_scripts/mosip_pms/ddl/pms-oidc_client.sql

```diff
@@ -0,0 +1,64 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-- -------------------------------------------------------------------------------------------------
+-- Database Name: mosip_pms
+-- Table Name : oidc_client
+-- Purpose    : oicd client: Table to store all registered OIDC client details.
+--           
+-- Create By   	: Nagarjuna K
+-- Created Date	: Sep-2022
+-- 
+-- Modified Date        Modified By         Comments / Remarks
+-- ------------------------------------------------------------------------------------------
+-- ------------------------------------------------------------------------------------------
+
+-- object: oidc_client.oidc_client | type: TABLE --
+-- DROP TABLE IF EXISTS oidc_client.oidc_client CASCADE;
+CREATE TABLE oidc_client(
+	id character varying(50) NOT NULL,
+	name character varying(256) NOT NULL,
+	rp_id character varying(50) NOT NULL,
```

Comment:
Why 50 it should be atleast 64 characters. I prefer we do 100 here so it can accommodate various future needs